internal/providers/terraform/google/testdata/secret_manager_secret_version_test/secret_manager_secret_version_test.golden

```diff
@@ -0,0 +1,29 @@
+
+ Name                                                                           Monthly Qty  Unit                    Monthly Cost 
+                                                                                                                                  
+ google_secret_manager_secret.secret_automatic                                                                                    
+ ├─ Active secret versions                                                Monthly cost depends on usage: $0.06 per versions       
+ ├─ Access operations                                                     Monthly cost depends on usage: $0.03 per 10K requests   
+ └─ Rotation notifications                                                Monthly cost depends on usage: $0.05 per rotations      
+                                                                                                                                  
+ google_secret_manager_secret.secret_example                                                                                      
+ ├─ Active secret versions                                                Monthly cost depends on usage: $0.06 per versions       
+ ├─ Access operations                                                     Monthly cost depends on usage: $0.03 per 10K requests   
+ └─ Rotation notifications                                                Monthly cost depends on usage: $0.05 per rotations      
+                                                                                                                                  
+ google_secret_manager_secret_version.secret_version                                                                              
+ ├─ Active secret versions                                                                2  versions                       $0.12 
```

Comment:
@aliscott In case when we have version resources and secret's `active_secret_versions` usage value specified, we might end up with an incorrect monthly cost. For example, the usage value is 10, and there are 2 version resources specified, it will show total cost for 12 versions. Should we warn about it anyhow?